### DISCUSSION TITLE
2.3.1

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#2617] [General]
## What's Changed
* fix: remove positive prompt from anime prefix by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2571
* fix: add enabled value to LoRA when setting default_max_lora_number by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2576
* fix: correctly set preset config and loras in meta parser by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2588
* fix: correctly load image number from preset / do not reset to 1 anymore by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2611
* fix: use correct base dimensions for outpaint mask padding by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2612
* fix: add compatibility for LoRAs in a1111 metadata scheme by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/2615

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/2.3.0...2.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lllyasviel/Fooocus/releases/tag/2.3.1'>2.3.1</a>.</em>

**Comment by mashb1t** [comment]
This release also marks the start of a 2-3 weeks break. I will still participate in open discussions but not monitor PRs / issues during this period.

**Comment by generativekey** [comment]
Thank u for the update! 👽Is the possibility of txt2vid on Fooocus roadmap?

**Reply by mashb1t** [reply]
txt2video is currently not on the roadmap

**Comment by flynetstudios** [comment]
Any chance of add compatibility with Zluda ??  This really push the performance on Radeon cards.

**Reply by mashb1t** [reply]
This is not planned, please use ROCm
https://github.com/lllyasviel/Fooocus/discussions/488#discussioncomment-8454380
